Unfolding/notebooks/README.md

```diff
@@ -0,0 +1,37 @@
+jssMeasurements
```

Comment:
I guess that some of the info here is outdated or simple a copy of the main README. 
Again, just as an exercise, because this is way more common than you think, remove the information before 1) and maybe include some information how to create a conda env from your yml file

Reply:
I've updated the readme, let me know if you think it needs any further additions.